54/hobojoe1848/main.py

```diff
@@ -0,0 +1,22 @@
+import pyperclip
+
+"""Keeping this simple, I needed to populate the clipboard_history list with
+a dummy entry to allow me to reference the 'last' index in the list in my print clipboard function"""
+
+clipboard_history = ["dummydata"]
+
+def print_clipboard():
+    """This function watches the clipboard and if anything NEW is added
+       it'll paste it to stdout."""
+    if pyperclip.paste() != clipboard_history[-1]:
+        clipboard_history.append(pyperclip.paste())
+        print(f"""\n{pyperclip.paste()}""")
+
+if __name__ == "__main__":
+    while True:
```

Comment:
you can use `try...except KeyboardInterrupt` to catch a cntrl+c to exit the program ...